packages/forklift-console-plugin/src/modules/Providers/views/create/components/OvirtProviderCreateForm.tsx

```diff
@@ -18,19 +18,22 @@ export const OvirtProviderCreateForm: React.FC<OvirtProviderCreateFormProps> = (
 
   const url = provider?.spec?.url || '';
 
-  const helperTextInvalid = {
+  const helperTextMsgs = {
     error: t(
-      'Error: Please provide a valid URL with a schema, domain, and path. For example: https://rhv.com/ovirt-engine/api.',
+      'Error: The provided URL is in an invalid format. Please provide a URL that includes a scheme, a domain name and path. For example: https://rhv-host-example.com/ovirt-engine/api',
     ),
     warning: t(
-      'Warning: The provided URL does not end with "ovirt-engine/api". Ensure it includes the correct path, like: https://rhv.com/ovirt-engine/api.',
+      'Warning: The provided URL does not end with the RHV API endpoint path: "ovirt-engine/api". Ensure it includes the correct path, For example: https://rhv-host-example.com/ovirt-engine/api',
```

Comment:
```suggestion
      'Warning: The provided URL does not end with the RHVM API endpoint path: "ovirt-engine/api". Ensure it includes the correct path, For example: https://rhv-host-example.com/ovirt-engine/api',
```

Reply:
Replace "it" with "the URL" or "The RHVM API endpoint path," whichever is correct.